colour/models/rgb/tests/tests_ycbcr.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Defines unit tests for :mod:`colour.models.rgb.ycbcr` module.
+"""
+
+from __future__ import division, unicode_literals
+
+import numpy as np
+import unittest
+
+from colour.models.rgb.ycbcr import (rgb_to_YCbCr, YCbCr_to_rgb,
```

Comment:
Would it be possible to get one import per line please if they don't fit all on the same one, cheers!